compiler/damlc/daml-opts/daml-opts/DA/Daml/Options.hs

```diff
@@ -470,6 +470,14 @@ expandSdkPackages lfVersion dars = do
             Just sdkPath -> pure $ sdkPath </> "daml-libs" </> fp <> sdkSuffix <.> "dar"
             Nothing -> fail $ "Cannot resolve SDK dependency '" ++ fp ++ "'. Use daml assistant."
       | otherwise = pure fp
+    -- For `dependencies` you need to specify all transitive dependencies.
```

Comment:
If we put the DALF for DAML Script into the DAR for DAML Triggers, wouldn't that solve the problem as well? If so, I would prefer that approach over having this custom hook here.

Reply:
No it doesn’t. We need the GHC interface files.